cloud/scope/object_storage_key.go

```diff
@@ -99,27 +99,7 @@ func (s *ObjectStorageKeyScope) AddFinalizer(ctx context.Context) error {
 	return nil
 }
 
-const (
-	accessKeySecretNameTemplate = "%s-obj-key"
-
-	ClusterResourceSetSecretFilename = "etcd-backup.yaml"
-	BucketKeySecret                  = `kind: Secret
```

Comment:
what are your thoughts on keeping this as a default in case type is resource-set so we don't have to add this into the templates?

Reply:
It makes more sense to me to keep it generic. There could be other times when secrets are generated in workload clusters for different use cases besides the one we define for etcd-backup-restore. Since we own the cluster resource-set defined in the etcd-backup-restore addon, it's easy for us to just define it there.

Reply:
very fair, sounds good to me!